lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -268,6 +268,20 @@ LazyNativeFunctions::convolution_backward_overrideable(
     const at::Tensor& weight, at::IntArrayRef stride, at::IntArrayRef padding,
     at::IntArrayRef dilation, bool transposed, at::IntArrayRef output_padding,
     int64_t groups, std::array<bool, 3> output_mask) {
+  // Lower to cudnn_convolution_backward when possbile
+  if (at::globalContext().userEnabledCuDNN() &&
+      lazy_tensors::sys_util::GetEnvBool("LTC_TS_CUDA", true)) {
```

Comment:
is LTC_TS_CUDA check necessary here? isn't there already an API in LTC which reads this variable and sets some default lazy device field?